river/XdgToplevel.zig

```diff
@@ -170,7 +170,7 @@ fn handleMap(listener: *wl.Listener(*wlr.XdgSurface), xdg_surface: *wlr.XdgSurfa
 
     view.surface = self.xdg_surface.surface;
 
-    // Use the view's "natural" size centered on the output as the default
+    // Use the views "natural" size centered on the output as the default
```

Comment:
The apostrophe was correct.
```suggestion
    // Use the view's "natural" size centered on the output as the default
```